package.json

```diff
@@ -70,6 +70,7 @@
     "react-polyglot": "^0.7.2",
     "react-qr-reader": "^2.2.1",
     "react-redux": "^7.2.6",
+    "react-router": "^5.2.1",
```

Comment:
This is to fix a hoisting issue because StoryBook uses 6.x and that version was being hoisted to the root of the project so I added an explicit dependency here to prevent that.